bump-crate.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
Oh and last one, typically it's better to declare these in a totally portable way
```suggestion
#!/usr/bin/env bash
```